.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org/
+@purecloud:registry=https://purecloud.jfrog.io/purecloud/api/npm/inin-internal-npm-dev/
```

Comment:
Did we want to expose this info into the public github?
Same with the jenkinsfile info

Reply:
It's not ideal, but it is safe since Neither file has any actual credentials in it. I added the .npmrc to help stop package-lock.json from continuously getting the PureCloud URLs in it (which I do constantly, since that's the default for my personal .npmrc, along with most folks in PureCloud)

I'm open to other suggestions, but I'd prefer not to keep build/doc stuff in a separate repo, since I suspect that will just mean it gets out of date quickly. 

I could maybe improve things  bit for potential external devs by making the deploy tools an optional dependency?